bpm_projects_api/core/security.py

```diff
@@ -0,0 +1,93 @@
+from datetime import datetime, timedelta
```

Comment:
A general question. Are you going to manage the authorization and authentication from a third service? 

This file looks like is not oriented to establish a connection with a third service.

Reply:
The integration with 3rd party services is not inside the scope of this PR. At the beginning many things will be hardcoded, giving priority to agility. An authentication and authorization server are not even defined as projects.

Reply:
But eventually, yes I think there should be an auth endpoint to generate the secret keys: for each user, for scope, or whatever the policy be. Each microservice as this one will have to ask this auth endpoint the JWT to access other services and those services will demand secret_keys every time they expires: the secret keys are used as signature to check if the token was really delivered by the auth endpoint. Does that makes sense?

Reply:
"An authentication and authorization server are not even defined as projects as far as I know, otherwise please let me know". 

Are you still unable to view the Context Map on draw.io? It's all right there.